keystore/kms/aws/encryptor.go

```diff
@@ -67,7 +65,7 @@ func (e *Encryptor) Encrypt(ctx context.Context, keyID string, data []byte) ([]b
 func (e *Encryptor) Decrypt(ctx context.Context, keyID string, blob []byte) ([]byte, error) {
 	input := &kms.DecryptInput{
 		CiphertextBlob: blob,
-		KeyId:          aws.String(keyID),
+		KeyId:          aws.String(fmt.Sprintf("alias/%s", keyID)),
```

Comment:
does AWS have doc page about the requirement to have `alias/` prefix for key ids? if yes, can we put reference here?

Reply:
Done